spec/services/processors/external/contributions_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe Processors::External::Contributions do
+  let!(:user) { create(:user, login: 'hvilloria') }
+  context 'when there are repos for the given username' do
+    before do
+      stub_get_repos_from_user(user.login)
+      allow(Builders::ExternalProject).to receive(:call) {}
+      allow(Processors::External::PullRequests).to receive(:call) {}
```

Comment:
why is everything stubbed?

Reply:
it was because of the external requests, but i changed it by request stub so all the flow could get completed 👍